cluster/addons/dns/coredns.yaml.base

```diff
@@ -94,6 +94,8 @@ spec:
     metadata:
       labels:
         k8s-app: kube-dns
+      annotations:
+        seccomp.security.alpha.kubernetes.io/pod: 'docker/default'
```

Comment:
Ideally, yes. But we recently updated it to runtime/default. So to handle the version skew, I think we have to use the old docker/default for now in case some clusters are still using an old version. Later when we enable default seccomp profile by default, this can be removed.

Reply:
Got it! Thanks for the explanation.